File: src/main/java/kz/chesschicken/astolfomod/block/ItemAstolfoStatue.java
package kz.chesschicken.astolfomod.block;

import net.minecraft.item.ItemInstance;
import net.modificationstation.stationapi.api.client.gui.CustomTooltipProvider;
import net.modificationstation.stationapi.api.template.item.TemplateBlock;

public class ItemAstolfoStatue extends TemplateBlock implements CustomTooltipProvider {
    
    public ItemAstolfoStatue(int i) {
        super(i);
    }

    @Override
    public String[] getTooltip(ItemInstance itemInstance, String originalTooltip) {
        return new String[]{
            "Astolfo Statue",
            "Statue: " + (itemInstance.getDamage() == 0 ? "Lightweight" : "Armored")
        };
    }
    
}

File: src/main/scala/kz/chesschicken/astolfomod/block/ItemAstolfoStatue.scala
package kz.chesschicken.astolfomod.block

import net.minecraft.item.ItemInstance
import net.modificationstation.stationapi.api.client.gui.CustomTooltipProvider
import net.modificationstation.stationapi.api.template.item.TemplateBlock

class ItemAstolfoStatue(i: Int) extends TemplateBlock(i) with CustomTooltipProvider {
  override def getTooltip(itemInstance: ItemInstance, originalTooltip: String): Array[String] = Array[String](
    "Astolfo Statue",
    "Statue: " + (if (itemInstance.getDamage == 0) "Lightweight" else "Armored")
  )
}
